include/fastdds/dds/topic/TopicDescription.hpp

```diff
@@ -65,21 +65,10 @@ class TopicDescription
         return type_name_;
     }
 
-    /**
-     * Get the TopicDescriptionImpl
-     *
-     * @return pointer to TopicDescriptionImpl
-     */
     virtual TopicDescriptionImpl* get_impl() const = 0;
 
 protected:
 
-    /**
-     * @brief Constructor using topic name and data type name
-     *
-     * @param name Name for the topic
-     * @param type_name Data type name
-     */
```

Comment:
I think this can be kept because being a protected method Doxygen does not include it in the API reference. Same applies to the destructor.

Reply:
I think the arguments have self explanatory names, so I don't think the doxydoc adds useful information